CoursesManager.MVVM/GlobalCache/GlobalCache.cs

```diff
@@ -134,7 +138,7 @@ private void EnsureCapacity()
 
     private void IncreaseCapacity()
     {
-        _capacity = _initialCapacity * 2;
+        _capacity = _initialCapacity + _capacity;
```

Comment:
Het is ook mogelijk om het volgende te doen:

```suggestion
        _capacity += _initialCapacity;
```